src/you_get/common.py

```diff
@@ -682,19 +690,21 @@ def __init__(self, total_size, total_pieces = 1):
     def update(self):
         self.displayed = True
         bar_size = self.bar_size
-        percent = round(self.received * 100 / self.total_size, 1)
-        if percent >= 100:
+        percent_str = round_str(self.received * 100 / self.total_size, 1)
+        if self.received == self.total_size:
             percent = 100
```

Comment:
To clarify, this is what it yields on [this video](https://www.youtube.com/watch?v=DnGdoEa1tPg):

![screenshot from 2017-07-22 23-34-27](https://user-images.githubusercontent.com/342945/28494699-717bf5ac-6f36-11e7-973b-f424529e11de.png)